app/assets/stylesheets/sightEdit.scss

```diff
@@ -5,6 +5,10 @@
     margin: 0 auto;
     color: $blue-purple;
 
+    h1{
```

Comment:
Probably best to use a classname unless you want all h1's in the application to use this font. If you do, you should move this style declaration to somewhere centralized like application.scss